/*---------------------------------------------------
Rodrigo Mungu√≠a 2021.

System parameters
-----------------------------------------------------
*/
#include <string>

#ifndef PARAMETERS_H
#define PARAMETERS_H

using namespace std;



struct parameters
{

    double run_time;
    string Dataset_path;
    double x_vel_run_time;
    bool restart;


} ;   





#endif /*PARAMETERS_H*/